config/database.pg.yml

```diff
@@ -15,7 +15,7 @@ base: &base
   adapter: postgresql
   encoding: utf8
   username: root
-  pool: 1
+  pool: 5

```

Comment:
@imtayadeway and I both needed to do this to get `rails s` to work under puma, even for a single request. (Because we access the database during app boot, though we really shouldn't.)